src/dotnet/commands/dotnet-new/CSharp_Web/$projectName$.csproj

```diff
@@ -11,16 +9,22 @@
     <PackageTargetFallback>$(PackageTargetFallback);portable-net45+win8+wp8+wpa81;</PackageTargetFallback>
```

Comment:
Correct, we add it to the base StarterWeb.csproj file in the vs templates since we don't have a way to conditionally inject this via a template engine rule and because this import should go away after we eventually move to the later runtime packages according to @prafullbhosale